src/screens/qr/views/NoCamera.tsx

```diff
@@ -15,36 +16,34 @@ export const NoCamera = ({updatePermissions}: {updatePermissions: () => void}) =
     updatePermissions();
   }, [updatePermissions]);
   return (
-    <Box backgroundColor="overlayBackground" flex={1}>
-      <SafeAreaView style={styles.flex}>
-        <Box marginBottom="m">
-          <Toolbar
-            title=""
-            navText={i18n.translate('RegionPicker.Close')}
-            navLabel={i18n.translate('RegionPicker.Close')}
-            navIcon="icon-back-arrow"
-            onIconClicked={close}
-          />
-        </Box>
-        <ScrollView style={styles.flex}>
-          <Box paddingHorizontal="m">
-            <Text variant="bodyTitle" marginBottom="l" accessibilityRole="header" accessibilityAutoFocus>
-              {i18n.translate('QRCode.CameraPermissions.Title')}
-            </Text>
+    <BaseQRCodeScreen showBackButton showCloseButton={false}>
+      <Box marginBottom="m">
+        <Toolbar
+          title=""
+          navText={i18n.translate('RegionPicker.Close')}
+          navLabel={i18n.translate('RegionPicker.Close')}
+          navIcon="icon-back-arrow"
+          onIconClicked={close}
+        />
+      </Box>
+      <ScrollView style={styles.flex}>
```

Comment:
Also I think this can be deleted since there is already a `ScrollView` in `BaseQRCodeScreen`